xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-xwiki/src/test/java/org/xwiki/rendering/internal/util/XWikiIconProviderTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.internal.util;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.xwiki.icon.Icon;
+import org.xwiki.icon.IconRenderer;
+import org.xwiki.icon.IconSet;
+import org.xwiki.icon.IconSetManager;
+import org.xwiki.localization.ContextualLocalizationManager;
+import org.xwiki.localization.LocalizationException;
+import org.xwiki.localization.Translation;
+import org.xwiki.localization.internal.DefaultTranslation;
+import org.xwiki.rendering.block.*;
+import org.xwiki.test.junit5.mockito.ComponentTest;
+import org.xwiki.test.junit5.mockito.InjectMockComponents;
+import org.xwiki.test.junit5.mockito.MockComponent;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+/**
+ * Unit tests for {@link XWikiIconProvider}.
+ *
+ * @version $Id
+ *  @since 16.2.0-rc1
```

Comment:
We normally don't put since versions on tests.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3023/commits/1ff7bb4f2591e05a3ae7b52b620a441b31f76f0c :+1: